about.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link rel="stylesheet" href="./css/about.css">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Conference</title>
+</head>
+
+<body>
+
+    <body>
+        <section class="navbar">
+            <div class="menu-icon">
+                <hr class="m-menu">
+                <hr class="m-menu">
+                <hr class="m-menu">
+            </div>
+        </section>
+        <ul class="menu-ul">
+            <img class="close" alt="close" src="./Images/close.png">
+            <li class="links link1"><a class="menu-links" href="#home">Home</a></li>
+            <li class="links link2"><a class="menu-links" href="./about.html">About</a></li>
+            <li class="links link3"><a class="menu-links" href="#program">program</a></li>
+        </ul>
```

Comment:
- [ ]  In about page in desktop version the mobile menu, it is for mobile in both desktop and mobile
- [ ] Also please add a link to home page in mobile menu, so you can navigate between both pages
- [ ] Consider to add a logo for your conference in the menu in left
- [ ] Please add top dark bar as in behance design
> behance design
![image](https://user-images.githubusercontent.com/53402600/175644080-b6ec0551-2d5c-4d1f-a3ed-c54d7c662693.png)